demo/README.md

```diff
@@ -0,0 +1,99 @@
+# Metrics Collector Demo Using Docker Images
+
+### In this simple Demo we will walk through enabling the new DSE Metrics Collector in a DataStax Enterprise (DSE) 6.7 Docker Container, exporting the metrics to a Prometheus Docker Container and visualizing these metrics with a Grafana Docker container.
+
+This Demo is created using a DSE 6.7, Prometheus 2.4.3 and Grafana 5.3.2 containers.
+
+## Prerequisites
+
+* A computer running MacOSX, Linux or a Linux Virtual Machine
+
+* Basic understanding of Docker images and containers. 
+
+* Docker installed on your local system, see [Docker Installation Instructions](https://docs.docker.com/engine/installation/). 
+
+* Docker Compose installed, if running linux see [Install Docker Compose](https://docs.docker.com/compose/install)
+
+* DataStax DSE Docker Images are hosted on [Docker Hub](https://hub.docker.com/r/datastax/dse-server/). For documentation including configuration options, environment variables, and compose examples head over to our official [Docker Docs](https://docs.datastax.com/en/docker/doc/index.html?utm_campaign=Docker_Cus_2019&utm_medium=web&utm_source=docker&utm_term=&utm_content=Web_DocsDocker)
+
+## Getting Started
+
+Lets start by creating a directory to work from.  From a CLI prompt create a directory in your home directory called insights and change to that directory
+
+
+```
+mkdir ~/insights && cd ~/insights
+```
+
+## Demo Data
+
+To get the demo data, we need to clone the [DataStax DSE Metrics Reporter GitHub](https://github.com/datastax/dse-metric-reporter-dashboards) reop with the following command
+
+```
+curl -Lo dse-metric-reporter-dashboards-master.zip https://github.com/datastax/dse-metric-reporter-dashboards/archive/master.zip && unzip dse-metric-reporter-dashboards-master.zip
+```
+
+Now we need to cd to the demo directory 
+
+```
+cd dse-metric-reporter-dashboards-master/demo
+```
+
```

Comment:
All the way from above comment to here isn't really required.  Just mention you first download the project